Packages/Tracking/Hands/Runtime/Scripts/HandBinder/HandBinder.cs

```diff
@@ -156,6 +156,11 @@ public override void BeginHand()
         /// </summary>
         void SetHandScale()
         {
+            if(!Application.isPlaying)
```

Comment:
I think this needs a different fix -> I want to be able to change scale and fingertip offsets at edit time, when there is a LSP in the scene and instantly see how the change affects the hand model in the scene.

Can you instead check whether there is an LSP in the scene and maybe also turn off editing of the offset values, if there isn't one?